hazelcast/src/test/java/com/hazelcast/test/starter/test/HazelcastStarterTest.java

```diff
@@ -56,4 +59,12 @@ public void testMemberWithConfig() {
         assertEquals(alwaysRunningMember.getName(), "test-name");
         alwaysRunningMember.shutdown();
     }
+
+    @Test
+    public void testGetOrCreateWorkingDir() {
+        File dir = HazelcastStarter.getOrCreateVersionDirectory("3.10-EE");
+        assertTrue(dir.exists());
+        // ensure no exception is thrown when attempt to recreate an existing version directory
+        HazelcastStarter.getOrCreateVersionDirectory("3.10-EE");
```

Comment:
Can we also check the returned `File` if it's the same path as the first method call?